ext_localconf.php

```diff
@@ -71,7 +71,7 @@ function includeRealurlConfiguration() {
 }
 
 // Exclude gclid from cHash because TYPO3 does not do that
-if (!in_array('gclid', $GLOBALS['TYPO3_CONF_VARS']['FE']['cacheHash']['excludedParameters'])) {
+if (is_array($GLOBALS['TYPO3_CONF_VARS']['FE']['cacheHash']['excludedParameters']) && !in_array('gclid', $GLOBALS['TYPO3_CONF_VARS']['FE']['cacheHash']['excludedParameters'])) {
     $GLOBALS['TYPO3_CONF_VARS']['FE']['cacheHash']['excludedParameters'][] = 'gclid';
     $GLOBALS['TYPO3_CONF_VARS']['FE']['cHashExcludedParameters'] .= ', gclid';
```

Comment:
@dmitryd `cHashExcludedParameters` is used for TYPO3 v7.6, and `['cacheHash']['excludedParameters']` for TYPO3 v8 as it looks to me?